cmd/nerdctl/volume_ls.go

```diff
@@ -127,5 +142,12 @@ func getVolumes(cmd *cobra.Command) (map[string]native.Volume, error) {
 	if err != nil {
 		return nil, err
 	}
+	withSize, err := cmd.Flags().GetBool("size")
+	if err != nil {
+		return nil, err
+	}
+	if withSize {
+		return volStore.ListWithSize()
```

Comment:
no need to create new func `ListXX` each time we add new column. `volStore.List(size)` ad check size value is enough